admin_pages/events/Event_Categories_Admin_List_Table.class.php

```diff
@@ -78,79 +76,114 @@ public function column_cb($item)
     }
 
 
-    public function column_id($item)
+    /**
+     * @param EE_Term $item
+     * @return string
+     * @throws EE_Error
+     * @throws ReflectionException
+     */
+    public function column_id(EE_Term $item)
     {
-        $content = $item->get('term_id');
-        $content .= '  <span class="show-on-mobile-view-only">' . $item->get_first_related('Term')->get(
-            'name'
-        ) . '</span>';
+        $category_name = $item->get_first_related('Term')->get('name');
+        $content       = $item->get('term_id');
+        $content       .= '  <span class="show-on-mobile-view-only">' . $category_name . '</span>';
```

Comment:
Do we need two spaces between term id and category name? Browsers will display only 1 space even if there are tens. We can use &nbsp; if you need more space or use CSS which is a lot better.

Reply:
good eye, no i'm guessing that extra space was just a typo. 
thnx will fix

Reply:
fixed